product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_biganimal_observability/index.mdx

```diff
@@ -58,7 +58,13 @@ By default, these charts displays the historical data of the last 15 minutes. To
   - **Active connections** (line chart) &mdash; The current number of connections between the client applications and the Postgres cluster.
   - **Disk throughput** (line chart) &mdash; The amount of data transferred to and from the disk per second for the Postgres cluster.
   - **Database size** (line chart) &mdash; The amount of storage volume used by the Postgres cluster.
+  - **Session wait states** &mdash; It is powered by [EDB wait states](https://www.enterprisedb.com/docs/pg_extensions/wait_states/) and provides comprehensive monitoring solutions for Postgres clusters. It actively monitors various metrics, including wait events, visualized through insightful charts. The chart supports prompt and informed actions. They include:
+    - **Wait times** (line chart) &mdash; This chart shows the approximate wait times for different wait events in a PostgreSQL database, measured in seconds. It groups each wait event type to make it easy to compare and identify potential system issues.
+    - **Wait events** (line chart) &mdash; This chart visualizes the approximate wait times of wait events in a PostgreSQL database, measured in seconds.
 
+    !!!note "Important"
+    The EDB wait states extension is enabled by default because it is needed to collect the data for Session wait states charts. If you want to choose not to view these data on charts, you can [disable the extension](https://www.enterprisedb.com/docs/pg_extensions/wait_states/installing/#uninstalling). This extension gathers detailed information about the active sessions, including wait events, which are then used to create helpful charts for diagnostics purposes. For more information on enabling/disabling the extension, see [EDB wait states](https://www.enterprisedb.com/docs/pg_extensions/wait_states/installing/).
```

Comment:
```suggestion
    The EDB Wait States extension is enabled by default because it is needed to collect the data for Session wait states charts. This extension gathers detailed information about the active sessions, including wait events, which is then used to create charts for diagnostics purposes. For more information on enabling/disabling the extension, see [EDB wait states](https://www.enterprisedb.com/docs/pg_extensions/wait_states/installing/).
```